.vscode/dryrun.log

```diff
@@ -0,0 +1,7 @@
+make --dry-run --always-make --keep-going --print-directory
+make: Entering directory '/home/varun/dev/go/src/cms.csesoc.unsw.edu.au'
```

Comment:
are the .vscode files necessary? it contains a path to your home directory

Reply:
ohh damm didnt notice htat